### PR TITLE
Make a new dummy play button

### PR DESCRIPTION
**Pull request by thatbirdguythatuknownot**
#7

### CODE REVIEW

**Comment by thatbirdguythatuknownot** [comment]
Linting errors are unrelated to the file changed.

**Comment by rustyxlol** [comment]
The check fails because of `isort`, changing the order of imports should fix it.

**Comment by thatbirdguythatuknownot** [comment]
> The check fails because of `isort`, changing the order of imports should fix it.

@rustyxlol #10 already fixed the lints and they're completely unrelated to #9.
I can't rebase this branch because I don't have `git` installed.

**Comment by rustyxlol** [comment]
Ah understood.

File: src/window/views.py [hunk 1]
```diff
@@ -48,6 +54,20 @@ def on_draw(self) -> None:
...
+        """
+        Do one of three things when the dummy play button is pressed.
+
+        [0] Open a URL using the `webbrowser` module. (The same module used by `import antigravity`)
+        [1] Close the window and exit python.
+        [2] Do both.
+        """
+        if (picked := randint(0, 2)) != 1:
+            webbrowser.open_new("https://youtu.be/fujCdB93fpw")
+        if picked:
```
Comment (Ibrahim2750mi) [inline src/window/views.py:67]: add `arcade.exit()` too while closing the app.

Comment (thatbirdguythatuknownot) [reply src/window/views.py:67]: Done.

**Review by Ibrahim2750mi — APPROVED** [review]

**Pull request merged**